### PR TITLE
Bug 1659514 Add reduced environment to telemetry/account-ecosystem

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1635659#c16

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] If the PR comes from a fork, trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [ ] Update `templates/include/glean/CHANGELOG.md`

### CODE REVIEW

File: schemas/telemetry/account-ecosystem/account-ecosystem.4.schema.json [hunk 1]
```diff
@@ -74,6 +170,9 @@
           },
           "type": "array"
         },
+        "reason": {
+          "type": "string"
+        },
```
Comment (jklukas) [inline schemas/telemetry/account-ecosystem/account-ecosystem.4.schema.json:175]: I also add payload.reason here, which is currently getting sent (and stored in additional_properties) and is quite standard for common pings.

File: templates/telemetry/pre-account/pre-account.4.schema.json [hunk 2]
```diff
@@ -15,84 +16,6 @@
       "minimum": 4,
       "maximum": 4
     },
-    "environment": {
```
Comment (jklukas) [inline templates/telemetry/pre-account/pre-account.4.schema.json:18]: The environment fields are the same as for the old pre-account ping, so I've factored out that existing definition to an include that now appears both here and in the new account-ecosystem ping.

**Comment by jklukas** [comment]
@acmiyaguchi I'm seeing in the generated BQ schema diff that many of the new fields are marked "REQUIRED", which doesn't seem right. Shouldn't our BQ schemas have only NULLABLE and REPEATED fields?

**Comment by acmiyaguchi** [comment]
@jklukas The CI used in the repo is slightly different from how we actually deploy it. We don't force all values to be nullable here, in particular (https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/7ea6c19b448f4785b2b43ad40229cb24c5de91e1/scripts/bigquery_schema_diff.py#L39-L53 vs https://github.com/mozilla/mozilla-schema-generator/blob/bafad8868988ed54ff57a57c0f063fbe903ada50/bin/schema_generator.sh#L171-L177).

**Pull request merged**